### DISCUSSION TITLE
Migration: nextcloud version issue

### DISCUSSION THREAD

**Discussion by rderooy** [nextcloud/all-in-one#3632] [Questions] [answered]
I am doing a migration from a local nextcloud installation to the Docker AIO image, but my nextcloud instance is currently at 27.1.0 and the latest release shown is 27.1.3. This is the only version that the UI allows me to upgrade to.

However the AIO version is still at 27.1.2.

Is there a way to upgrade my local installation to a specific release, to match the AIO version, such that I can continue with the migration?

**Comment by szaimen** [comment, marked as answer]
Hi, AIO v7.5.1 was just released which includes Nc 27.1.3. See https://github.com/nextcloud/all-in-one/releases/tag/v7.5.1
